integration-test/tests/performance.extension.js

```diff
@@ -0,0 +1,112 @@
+/**
+ *  Performance testing
+ */
+import { setup } from '../helpers/harness.js'
+import ss from '../../src/Form/test-cases/index.js'
+import * as fs from 'fs'
+const baseline = JSON.parse(fs.readFileSync('./integration-test/stats/scan-original.json', 'utf8'))
+
+xdescribe('Performance measuring', () => {
```

Comment:
Just double checking that this `xdescribe` is intentional. I suppose we only want to run the performance checks on demand and not in CI.

Reply:
Yep - the tests will be relative to the machine on which they run, so are not good signals to use in CI